Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,18 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+console.log(tasks);
```

Comment:
once you have finished the exercise, try to remove unnecessary console.logs. it's make your code more clean

Reply:
I will remember that, thank you.